README.md

```diff
@@ -157,6 +157,106 @@ Gets the permission information of a record
 
 ---
 
+# Append data to Linnia
+
+In order to append data to Linnia your web3 instance need to be able to sign with the private key of the owner of the data.
+
+## linnia.addRecord
+
+```javascript
+linnia.addRecord(dataHash, metadata, dataUri, ethParams);
+```
+
+Add record to Linnia
+
+### Parameters
+
+1. `String` - The data hash. Hash of the plain text data + metadata
+2. `String|Object` - The metadata of the record. [Click here to read more about the metadata](https://github.com/ConsenSys/linnia-resources/blob/master/METADATA.md)
```

Comment:
Maybe we should only accept objects here if we strongly recommend json?

Reply:
sure, I like it